dashboard/src/icons/kubeapps.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="200" viewBox="55 45 520 520">
```

Comment:
I'd say the rest of the icons are defaulted to 100x100. Have a look at the rest of the svg files (I should have wrote down the tool names I used)

Reply:
The `width` parameter is not really relevant. It is just the defaulting size used by rendering tool (i.e. browser).
As it is a vectorial image, whatever size it is set by CSS will prevail, and the image will be beautifully rescaled without glitches.

Reply:
Have you used any tool that rewrites the vectorial path in a smaller view box?

Reply:
Yep, I used the same view box, with the actual icon centered and a fixed height/width.

Even if the visualization can be later changed with css easily, it is useful when placing several svg and applying the same style to all of them at once.

For instance, I'd recall the Carvel and helm icons being misplaced (due to the canvas size and the icon not being centered) when rendering the catalog cards.

Reply:
Also, I've noticed this is the b&w image, not the colored one. Shouldn't we use (or add it in addition to this one) the other?

Reply:
Couldn't find the coloured SVG of the logo. Happy to add that one in addition.